spec/controllers/users_controller_specs.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::V1::UsersController, type: :controller do
+  before do
+    request.headers['ACCEPT'] = 'application/json'
+  end
+
+  describe 'DELETE users#destroy' do
+    it 'deletes the user' do
+      user = create(:user)
+      delete :destroy, params: { id: user.id }
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'returns an error if the user id is invalid' do
+      delete :destroy, params: { id: 0 }
```

Comment:
0 could be a valid id at some point. I would change it to nil, or something like `id: "invalid-id"`